actors/builtin/miner/miner_test.go

```diff
@@ -1353,6 +1366,37 @@ func TestDeclareFaults(t *testing.T) {
 	})
 }
 
+func TestDeclareRecoveries(t *testing.T) {
```

Comment:
This is great. Since your at it, a quick happy path test of declare recoveries would be nice. Should be the the same test without the IP Debt part and a state assertion that the sector has been declared recovered.